tests/e2e/test-site/autoscript.js

```diff
@@ -0,0 +1,91 @@
+(function () {
+  const rxFlatMap = rxjs.operators.flatMap;
+  const rxMap = rxjs.operators.map;
+  const rxTap = rxjs.operators.tap;
+  const url = new URL(window.location.href);
+  const clientId = url.searchParams.get('clientId');
+  const scope = url.searchParams.get('scope');
+  const tenant = url.searchParams.get('tenant');
+  const tree = url.searchParams.get('tree');
```

Comment:
`tree` should probably be hard-coded here to "UsernamePassword", otherwise the callbacks can be unpredictable.  IOW, these tests might need to be aligned to known authentication trees shipped in Express.